index.html

```diff
@@ -6,15 +6,14 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
```

Comment:
[ OPTIONAL]
- [ ] Kindly consider adding a descriptive title
`hint: name's portfolio`
- [ ] Kindly make sure all your `alt`  attributes have a descriptive name telling what the image is.